docs/user-guide.md

```diff
@@ -417,7 +558,8 @@ TODO
 import org.graphframes.examples
 val g: GraphFrame = examples.Graphs.friends  // get example graph
 
-TODO
+val result = g.labelPropagation.maxSteps(5).run()
+result.vertices.select("id", "label").show()
```

Comment:
Maybe not part of this PR. It would be easy to understand the algorithm if we attach the result. Btw, `show` should be called with `truncate=false`, or users won't see the full output.